explorer/explorertypes.go

```diff
@@ -304,18 +305,31 @@ type MempoolInfo struct {
 
 // MempoolShort represents the mempool data sent as the mempool update
 type MempoolShort struct {
-	LastBlockHeight    int64          `json:"block_height"`
-	LastBlockTime      int64          `json:"block_time"`
-	TotalOut           float64        `json:"total"`
-	TotalSize          int32          `json:"size"`
-	NumTickets         int            `json:"num_tickets"`
-	NumVotes           int            `json:"num_votes"`
-	NumRegular         int            `json:"num_regular"`
-	NumRevokes         int            `json:"num_revokes"`
-	NumAll             int            `json:"num_all"`
-	LatestTransactions []MempoolTx    `json:"latest"`
-	FormattedTotalSize string         `json:"formatted_size"`
-	TicketIndexes      map[string]int `json:"ticket_indexes"`
+	LastBlockHeight    int64                    `json:"block_height"`
+	LastBlockHash      string                   `json:"block_hash"`
+	LastBlockTime      int64                    `json:"block_time"`
+	TotalOut           float64                  `json:"total"`
+	TotalSize          int32                    `json:"size"`
+	NumTickets         int                      `json:"num_tickets"`
+	NumVotes           int                      `json:"num_votes"`
+	NumRegular         int                      `json:"num_regular"`
+	NumRevokes         int                      `json:"num_revokes"`
+	NumAll             int                      `json:"num_all"`
+	LatestTransactions []MempoolTx              `json:"latest"`
+	FormattedTotalSize string                   `json:"formatted_size"`
+	TicketIndexes      map[int64]map[string]int `json:"ticket_indexes"`
```

Comment:
I do not believe this will marshal to JSON.  Probably will be something like `json: unsupported type: map[int64]...`.

Since this is only being used for the go html template processing, not JSON marshaling (presently), I suggest `json:"-"` for this